frontend/src/utils/loadingStatus.ts

```diff
@@ -0,0 +1,6 @@
+export enum loadingStatus {
+  LOADING = 'LOADING',
+  UNLOADED = 'UNLOADED',
```

Comment:
what is UNLOADED meant to represent?

Reply:
UNLOAD means not loaded yet LOL.Though, this doesn't make any sense so I changed it to `NOT_LOADED`